config/routes.rb

```diff
@@ -3,4 +3,31 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # CREATE Tweets
+  resources :tweets, only: [:new, :create]
+   # UPDATE Tweets
+  resources :tweets, only: [:edit, :update]
```

Comment:
Is not necesary to call it twice you can do it just by using it as in the suggestion
```suggestion
  resources :tweets, only: [:new, :edit, :create, :update]
```